### DISCUSSION TITLE
Which companies use V?

### DISCUSSION THREAD

**Discussion by bistcuite** [vlang/v#12398] [General]
Hi, which companies use V?

**Comment by JalonSolov** [comment]
I don't know if any companies use V, yet.  V is still Alpha software, and most companies want something more stable.

**Comment by nedpals** [comment]
Will be converting this into a discussion for these types of questions.

**Comment by chriseaton** [comment]
Probably too early for stable commercial popularity, but plenty of devs here working for and leading companies that are interested in V's future, including me.
